### PR TITLE
Added banner notice for documentation move

### PR DESCRIPTION
**Pull request by owenmerry**
Screenshot 2022-10-17 at 14 05 57

### CODE REVIEW

**Review by apavlidi — APPROVED** [review]
Looks good 👏

**Review by ss-sarak — APPROVED** [review]

**Comment by ss-sarak** [comment]
I guess since we're deprecating it, we don't care about a lot about the stylings/Skyscanner color palette, right? 

LGTM 👍

File: source/stylesheets/screen.css.scss [hunk 1]
```diff
@@ -58,6 +71,7 @@ html, body {
 }
 
 .toc-wrapper {
+  padding-top: 60px;
```
Comment (apavlidi) [inline source/stylesheets/screen.css.scss:74]: Are we concerned about responsiveness in our slate docs? I haven't tried it locally but I just saw this hard-coded px and thought about this ☺️

**Comment by owenmerry** [comment]
"I guess since we're deprecating it, we don't care about a lot about the stylings/Skyscanner color palette, right?"

Yep just picked a random red.. the whole site isn't on theme so all good

File: source/stylesheets/screen.css.scss (hunk 1 above)
Comment (owenmerry) [reply source/stylesheets/screen.css.scss:74]: I have checked it on mobile to make sure it didnt break it, doubt much mobile traffic but wont stop mobile users

**Pull request merged**